### PR TITLE
Update bootstrap.css bootstrap version updated

### PR DESCRIPTION
**Pull request by prateekiiest**
issue #39

### CODE REVIEW

**Comment by prateekiiest** [comment]
Some of the new features still needed to be incorporated

File: js/bootstrap.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*!
- * Bootstrap v3.0.3 (http://getbootstrap.com)
+ * Bootstrap v3.3.7 (http://getbootstrap.com)
```
Comment (dpshelio) [inline js/bootstrap.js]: There must be something wrong here, this file only changes the comments of the version? I've seen other PRs with new version of bootstrap and it has a larger amount of changes.

Comment (prateekiiest) [inline js/bootstrap.js]: yes, the functionality also needs to be changed in some of the functions.
I'm working on it

**Comment by ehsteve** [comment]
this PR seems to be conflicting with updates from #73

**Comment by prateekiiest** [comment]
Oh, someone has already done this.
 @ehsteve @dpshelio 
Will I close this then or work on it?

**Pull request merged**